search/includes/classes/class-search.php

```diff
@@ -1955,4 +1958,15 @@ public function log_ep_invalid_response( $request, $query ) {
 
 		$this->logger->log( 'warning', 'search_query_failure', $message );
 	}
+
+	/**
+	 * Filter out default algorithm version to be used.
+	 * 
+	 * @param $default_algorithm_version string Algorithm version.
+	 * 
+	 * @return string
+	 */
+	public function filter__ep_search_algorithm_version( $default_algorithm_version ) {
+		return '3.5';
```

Comment:
The PR description says to lock to 3.4, yet this uses `3.5`?

Reply:
@GaryJones I should have updated the PR description (which I will do now!) — after some discussion with @rinatkhaziev, we're going to lock it at 3.5 to futureproof any surprises